samza-core/src/main/scala/org/apache/samza/container/SamzaContainerMetrics.scala

```diff
@@ -47,6 +47,7 @@ class SamzaContainerMetrics(
   val diskQuotaBytes = newGauge("disk-quota-bytes", Long.MaxValue)
   val executorWorkFactor = newGauge("executor-work-factor", 1.0)
   val physicalMemoryMb = newGauge[Double]("physical-memory-mb", 0.0F)
+  val containerMemoryUtilization = newGauge("container-memory-utilization", 0.0F)
```

Comment:
1. Can we be more specific and call this `physical-memory-utilization`?
2. Can you update the [monitoring documentation](https://github.com/apache/samza/blob/master/docs/learn/documentation/versioned/operations/monitoring.md) as well 
3. Do we need to explicitly pass the metric value type here? I see we have inconsistencies above where some pass the type `Double` and some don't. Will be good to maintain consistency one way or the other.

Reply:
1. Sure
2. Done
3. Fixed the `Double` ones to be like the others, i.e. use `Float` as the type instead